roottest/root/io/transient/base/CMakeLists.txt

```diff
@@ -1,25 +1,27 @@
-
-include_directories(${CMAKE_CURRENT_SOURCE_DIR})
-set(CMAKE_ROOTTEST_NOROOTMAP OFF)
-ROOT_GENERATE_DICTIONARY(base_cling ${CMAKE_CURRENT_SOURCE_DIR}/base.h MODULE base LINKDEF baseLinkDef.h)
-ROOTTEST_LINKER_LIBRARY(base base_cling.cxx testobject.cpp testobjectderived.cpp TEST LIBRARIES ROOT::Core ROOT::RIO)
-
-set(RootExeOptions -e "gSystem->Load(\"libbase\")")   # Adding a ; to avoid the output is complicated
+ROOTTEST_GENERATE_DICTIONARY(libRoottestIoTransientBase
+                             testobject.h testobjectderived.h
+                             LINKDEF baseLinkDef.h
+                             SOURCES testobject.cpp testobjectderived.cpp
+                             NO_CXXMODULE
```

Comment:
This seems to be a behavior change (vs `MODULE base`).  Why is it advantageous?

Reply:
I had failures on some platforms - because of auto loading with hadd

Reply:
We should add a comment here of what those failures are and why they require the no cxxmodule.  However I also don't see immediately why it worked before and now fails.